src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -9,6 +9,7 @@
  * elevation to support local machine remoting).
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-windows`
Tests for $ErrorView.NormalView tests.Error shows up when using strict mode
```powershell
Expected like wildcard '*Oops!*' to match '
', but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Formatting\ErrorView.Tests.ps1: line 82
82:             $e | Should -BeLike '*Oops!*'
```